lib/cli-engine.js

```diff
@@ -496,15 +517,19 @@ CLIEngine.prototype = {
             results.push(res);
         }
 
-        files.forEach(function(pattern) {
-            glob.sync(pattern, globOptions).forEach(executeOnFile);
+        patterns.forEach(function(pattern) {
+            if (shell.test("-f", pattern) && !ignoredPaths.contains(pattern)) {
```

Comment:
What do i need to have in my eslintignore file to make these 2 conditions true
- `shell.test("-f", pattern)` === true
- `ignoredPaths.contains(pattern)` === true